crates/bevy_internal/Cargo.toml

```
@@ -134,6 +134,11 @@ pbr_anisotropy_texture = [
   "bevy_gltf?/pbr_anisotropy_texture",
 ]
 
+# Percentage-closer soft shadows
+pbr_pcss = [
+  "bevy_pbr?/pbr_pcss",
+]
```

```suggestion
pbr_pcss = ["bevy_pbr?/pbr_pcss"]
```

This should be enough to make CI happy